### ISSUE TITLE
Plugin for okular

### ISSUE THREAD

**Issue by simoncarrignon** [chetachiezikeuzor/Yin-and-Yang-Theme#105]
I may have missed it but it would be nice to have a plugin for Okular  (KDE defautl pdf reader) to invert pdf's colors (and avoid all this white!) .
Okular's accessibility options allow to invert color, thus forcing a dark mode.  I think it should be  easy to do it by modifying Okular's config files (/usr/share/config.kcfg/okular.kcfg ), and I may try to do it myself if I have time! 
Edit: a few more info [here](https://www.reddit.com/r/kde/comments/ljutsj/guide_enable_a_dark_reader_in_okular)
